src/components/utils/test/qdb_wrapper/start_test_qdb_server.sh

```diff
@@ -11,10 +12,14 @@ fi
 if [ ! -d $DB_BACKUP_PATH ] ; then
    mkdir -p $DB_BACKUP_PATH
 fi
-# Creating policy for the resumption
+# Creating policy for the policy
```

Comment:
@BSolonenko policy for the policy?

Reply:
@mkorniichuk  fixed in https://github.com/Ford-Luxoft/sdl_core/pull/30/commits/d2cf4a58dff4be7458458a5b23d94e0c0ecc9d0f